<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Tryby</string>
    </property>
    <widget class="QPushButton" name="normal">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>NORMAL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="screen">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>SCREEN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lighten">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>LIGHTEN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="multiply">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>MULTIPLY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="darken">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>DARKEN</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Warto≈õci</string>
    </property>
    <widget class="QSlider" name="img1Slider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>18</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="img2Slider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>18</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="img3Slider">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>18</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="img1SB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="img2SB">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="img3SB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>48</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybrane Zdjecie</string>
     </property>
    </widget>
    <widget class="QLabel" name="ImageLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nazwa/Tryb:</string>
     </property>
    </widget>
    <widget class="QLabel" name="nazwaLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Zdjecie</string>
     </property>
    </widget>
    <widget class="QLabel" name="trybLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tryb</string>
     </property>
    </widget>
   </widget>
   <widget class="alphaBlending" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Zdjecia</string>
    </property>
    <widget class="QPushButton" name="Image1Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>IMG1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Image2Button">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>IMG2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Image3Button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>61</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>IMG3</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>alphaBlending</class>
   <extends>QWidget</extends>
   <header location="global">alphablending.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
